OpenRA.Game/Graphics/Theater.cs

```diff
@@ -68,6 +68,9 @@ public Theater(TileSet tileset)
 					var indices = t.Value.Frames != null ? t.Value.Frames : Enumerable.Range(0, frameCount);
 					variants.Add(indices.Select(j =>
 					{
+						if (j >= allFrames.Length)
+							throw new InvalidDataException(i + " defined more frames than the sprite has on template id " + t.Key);
```

Comment:
Derp, I forgot to mention that this should use the `{0}` and `{1}` syntax.

Reply:
Hmmkay